maths/montecarlo.py

```diff
@@ -49,7 +50,7 @@ def area_under_line_estimator(iterations: int) -> float:
     5. Returns estimated value
     """
     
-    estimate = sum(uniform(0,1) for i in range(iterations)) / iterations
+    estimate = mean(uniform(0,1) for i in range(iterations))
     
     return estimate
```

Comment:
We can just `return mean(uniform(0,1) for i in range(iterations))` because the function name already tells us that this is an estimate.

Reply:
Let’s change i to _ to signal that we are not using this value.

Reply:
Thanks. will make the change.

Reply:
Also, i think `uniform(0,1)` would fail **black** checks